### DISCUSSION TITLE
Share your awesome QR code designs with others!

### DISCUSSION THREAD

**Discussion by alexzhirkevich** [alexzhirkevich/qrose#1] [Show and tell]
Here you can post images of your awesome QR code designs with its source code.

Please use  images of 300x300 resolution and collape your code in a `details` markdown like below:

```markdown

<img src="your image" width="300">

<details>
<summary>Source code</summary>

```kotlin
val yourCodeGoesHere = 
 ```
```

<details>
<summary>Source code</summary>

```kotlin
val yourCodeGoesHere = 
```

**Comment by rafaelekol** [comment]
Can you add here some examples from your other branch 
https://github.com/alexzhirkevich/custom-qr-generator/tree/main/examples/instagram
